### ISSUE TITLE
mobilenet_v2 pascal voc  model and weight are mismatched

### ISSUE THREAD

**Issue by jorsua133** [Andrew-Qibin/ssdlite-pytorch-mobilenext#6]
I try to use pretrained weigth of mobilenet_v2 pascal voc and test using the given config file. However, it seems like the model and weight are mismatch. I have tried to manually adjust the channel numbers of extra layers to match the weight but accuracy is not correct.
![image](https://user-images.githubusercontent.com/99957209/226835136-2330241d-9881-441f-9b1f-d4e690212a3f.png)
ed.

**Comment by nicobh15** [comment]
Hey, have you found a solution to this? I am facing the same issue.

**Comment by jorsua133** [comment]
> Hey, have you found a solution to this? I am facing the same issue.

I retrained the model
